src/test/resources/data/InstructorStudentListPageUiTest.json

```diff
@@ -230,7 +230,7 @@
       "name": "Alice Betsy</option></td></div>'\"",
       "comments": "This student's name is Alice Betsy</option></td></div>'\"",
       "team": "Team 1</td></div>'\"",
-      "section": "Section A"
+      "section": "Section A</td></div>'\""
```

Comment:
Our current code too has cases like this. The slight downside of writing test cases like this is that the UI now looks like its broken to the naked eye. i.e. we don't expect to see this kind of section names unless there is a bug somewhere. Once we get desensitized to seeing garbage text in the UI, even real bugs can be ignored because we don't know which garbage is intentional and which is not. So it helps if you go the extra mile to indicate that this is not a bug. e.g. `Section A</td>html tags should appear here</div>`
Another thing we can do is to limit using garbage values to tests in which we clearly expect to see such 'seemingly garbage' values. i.e. not to mix sanitization testing with tests that are meant to test other things.